obal/data/roles/setup-sources/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- set_fact:
+    pkg_dir: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
+    annex_opts: annex_opts
```

Comment:
I can live with the shortening of words if others are, but generally I tend towards explicit verbosity (e.g. package over pkg and options over opts).

Reply:
Does this need a tag always?